recipes/paramz/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "paramz" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "d60a6449ce238dae6183e87a0f203077c8a463315c17f267e065b173bf5795e4" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - six
+    - decorator
+test:
+  imports:
+    - paramz
+
+about:
+  home: https://github.com/sods/paramz
+  license: BSD 3-clause
```

Comment:
please manually include the license file:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually